src/controller/storagemgr/storage.go

```diff
@@ -0,0 +1,46 @@
+package storagemgr
+
+import (
+	"controller/storagemgr/storagedriver"
+	"errors"
+	"github.com/edgexfoundry/device-sdk-go"
+	"github.com/edgexfoundry/device-sdk-go/pkg/startup"
+	"log"
```

Comment:
Please use the [common/logmgr](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/doc/logmgr.md) instead of the package log.

Reply:
Ok. I will do that.